src/main/scala/org/bitcoins/core/protocol/transaction/TransactionConstants.scala

```diff
@@ -29,7 +31,7 @@ trait TransactionConstants {
     * is set, the relative lock-time has units of 512 seconds,
     * otherwise it specifies blocks with a granularity of 1.
     */
-  def sequenceLockTimeTypeFlag = (1L << 22)
+  def sequenceLockTimeTypeFlag = 1L << 22
```

Comment:
This should be a UInt32 
https://github.com/bitcoin/bitcoin/blob/d612837814020ae832499d18e6ee5eb919a87907/src/primitives/transaction.h#L81